pkgs/applications/misc/megacmd/default.nix

```diff
@@ -8,7 +8,7 @@
   # build fails with latest ffmpeg, see https://github.com/meganz/MEGAcmd/issues/523.
   # to be re-enabled when patch available
   # , ffmpeg
```

Comment:
Why not to use `enable_XXX` Boolean guard attrs?

Reply:
Seems like a pretty niche use-case, as it will not build with ffmpeg. Also, I'm not sure the ffmpeg features of the sdk actually do anything in megacmd.

Reply:
Well, it would help to document the intentions of those "debug-hide-comments". 
You would set something like 

```nix
meta.broken = enable_ffmpeg || enable_freeimage; # insecure 
```

Reply:
The freeimage build isn't broken, just insecure. It might be better to just delete the comments entirely, upstream don't seem very interested in fixing the build.